linux/usage/root.md

```diff
@@ -18,6 +18,6 @@ You can also run a superuser shell by using `sudo su`. When running commands as
 
 ## Who can use sudo?
 
-It would defeat the point of the security if anyone could just put `sudo` in front of their commands, so only approved users can use `sudo` to gain administrator privileges. The `pi` user is included in the `sudoers` file of approved users. To allow other users to act as a superuser you can add the user to the `sudo` group with `usermod`, edit the `/etc/sudoers` file, or add them using `visudo`.
+It would defeat the point of the security if anyone could just put `sudo` in front of their commands, so only approved users can use `sudo` to gain administrator privileges. The `pi` user is included in the `sudoers` file of approved users. To allow other users to act as a superuser, add the user to the `sudo` group with `usermod`.
```

Comment:
> add the user to the `sudo` group with `usermod`.

Maybe it's worth providing an example command of how you'd do that?

Reply:
I would rather not get bogged down with adding more documentation about sudo to that page. That paragraph at least gives the reader a starting point - it's not intended to give detailed instructions. They can ask on the forums, or read the man page (linux/usage/commands.md tells them about 'man').